packages/components/src/utils/colors-values.js

```diff
@@ -31,9 +31,9 @@ const ALERT = {
 
 // Matches @wordpress/base-styles
 const ADMIN = {
-	theme: 'var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba))',
+	theme: 'var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9))',
 	themeDark10:
-		'var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #006ba1))',
+		'var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #2145e6))',
```

Comment:
We actually have a "dark blueberry" at #1D35B4, perhaps you can use that? It's from [here](https://www.figma.com/file/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?type=design&node-id=28593%3A26957&t=V7FzH6mnWqdOH4gW-1).

Reply:
So, these darkened Blueberry colors come from the actual values that are used in the "Modern" admin color scheme right now (selectable at `/wp-admin/profile.php`).

<img width="254" alt="Color scheme CSS variables in devtools" src="https://user-images.githubusercontent.com/555336/235448437-c465b2ec-8777-4575-813e-70766e78d5b8.png">

The entire color scheme is generated automatically (based on a single primary color) in `@wordpress/base-styles`:

https://github.com/WordPress/gutenberg/blob/d80ecc277efd2a8e76c17fc059362a92b7437e37/packages/base-styles/_mixins.scss#L417-L420

It would be easier to maintain if we treated these autogenerated values as canonical, especially because there are also 8 other admin color schemes. But we can override values manually if you think it's worth it. What is your preference?

Reply:
Ah, no no we can keep those existing values for now. I just saw a hard-coded value here and was confused as to the source. 

It's a bit of a chicken and egg thing — we have a human created color palette that we can hopefully use to inform the auto-generation system, so that shades of other hues can be generated from that original set of shades. In some cases that won't work, and we'll likely have to update the Figma palettes to match what the automatic system does.

If we keep this in mind as something to revisit in the future, no need to add complexity to this PR.

Reply:
Makes sense, thank you!